packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -249,7 +248,7 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
   contact: {
     extensions: ['vcf']
   },
-  css: {}, // we dont have the icon yet, but i believe we want it, snapping to 'code' for now
+  /*  css: {},  not broken out yet, snapping to 'code' for now */
```

Comment:
Technically it's a breaking change to remove the `css` and `channelfolder` properties, even if they weren't being used. I'm not familiar with how the file type icons work--would it cause any problems for those keys to continue to be present but empty?

Reply:
Should have said, it's a breaking change if this const is exported from the package. If it's not exported from the root, the change is probably fine.

Reply:
it's a good question. channelfolder is never used. css is covered in another row, so these changes wont affect any given filetype icon rendering in the current product.

Reply:
@ecraig12345 @maxwellred let me know if this is cool and we can merge.